src/ios/CDVBarcodeScanner.mm

```diff
@@ -623,6 +624,7 @@ - (NSString*)formatStringFromMetadata:(AVMetadataMachineReadableCodeObject*)form
     if (format.type == AVMetadataObjectTypeCode128Code)     return @"CODE_128";
     if (format.type == AVMetadataObjectTypeCode93Code)      return @"CODE_93";
     if (format.type == AVMetadataObjectTypeCode39Code)      return @"CODE_39";
+    if (format.type == AVMetadataObjectTypeInterleaved2of5Code) return @"ITF";
     if (format.type == AVMetadataObjectTypeITF14Code)          return @"ITF";
```

Comment:
Now "ITF" is twice here.
I think yours is fine, but the old one should be changed to "ITF_14".
Can you change that and also fix the merge conflicts?